lib/fluent/plugin/out_bigquery.rb

```diff
@@ -143,6 +143,10 @@ def configure_for_load(conf)
     config_param :request_timeout_sec, :time, default: nil
     config_param :request_open_timeout_sec, :time, default: 60
 
+    ## Partitioning
+    config_param :time_partitioning_type, :string, default: nil
+    config_param :time_partitioning_expiration, :integer, default: nil
```

Comment:
I think that `:time` type is better than `:integer`.
`:time` can accept `1m`, `1h`, or `1d` format. it is more readable.